specs/APIReview_UAString

```diff
@@ -0,0 +1,149 @@
+The purpose of this spec is to describe a new WebView2 feature - Support for UA Strings - and its APIs.
```

Comment:
For an initial spec review you review with me and you do a PR from a branch like api-uastring-draft to api-uastring. After that you do a PR from api-uastring to master and that we'll use for review with API board and customers on github

Reply:
Ah, my bad, this was meant to be the api-uastring-draft, after we get the review, maybe I can create another branch 'api-useragentstring' as the one we can review w/ api board & customers @david-risney - would this be ok?

Reply:
The name of the branch isn't a big deal. Its more important to not go straight to master:

PR1, branch A -> branch B, we review together
PR2, branch B -> master, we review with API review board

Can you change this PR to not go to master?

Reply:
Took care of it, sorry for the confusion!